editor/editor_help_search.cpp

```diff
@@ -588,7 +599,13 @@ TreeItem *EditorHelpSearch::Runner::_create_method_item(TreeItem *p_parent, cons
 		}
 	}
 	tooltip += ")";
-	return _create_member_item(p_parent, p_class_doc->name, "MemberMethod", p_doc->name, p_text, TTRC("Method"), "method", tooltip);
+
+	if (p_is_annotation) {
+		// TODO: Different icon?
```

Comment:
Yeah it should likely have an icon derived from `@`.

Reply:
Having just the `@` looks a bit weird when doubled with the annotation's name... 

![image](https://user-images.githubusercontent.com/5117197/187979127-9f7d6025-08f2-4329-a240-8490f8f7e02d.png)

I'm not a graphics designer, so I wouldn't mind if someone else comes up with a better icon :)

Reply:
It's fine for now